scripts/i18n/package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Scripts for gatsby internationalization",
   "scripts": {
     "create": "node ./create.js",
+    "run-all": "node ./run-all.js",
```

Comment:
Shouldn't this pass name of script as extra argument?

`run-all.js` seems to expect it

Reply:
You would run it as `yarn run-all [script-name]`. I've run it and it works, but is there a convention for specifying that a script takes in an argument?

Reply:
There is no convention for npm scripts I'm aware of, but given that `yarn run-all [script-name]` is expected usage we could provide more actionable message in case script-name is missing:
```diff
-logger.error(`Please provide a script name`)
+logger.error(`Usage: yarn run-all <script-name>`)
```
Above suggestion is what happens when CLI is implemented with `yargs` (so for example `gatsby`). `git`, or `npm` (without actual command) is similar etc (just bit more verbose because it has tons more options).

Would also be nice to have one-liner in README.md - other commands are documented there